pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -2,238 +2,198 @@
 
-The `venia-concept` storefront uses a newer version of Node than the default environment provides.
-It also uses Yarn to install its dependencies.
+```text
+variables:
+    env:
+        CONFIG__DEFAULT__WEB__UPWARD__PATH: "/app/pmu35riuj7btw_stg/pwa/upward.yml"
+        NODE_ENV: "production"
+        MAGENTO_BACKEND_URL: "https://[your-cloud-url-here]/"
+        CHECKOUT_BRAINTREE_TOKEN: "<generated token from Braintree>"
+        MAGENTO_BACKEND_EDITION: "EE"
+        IMAGE_OPTIMIZING_ORIGIN: "backend"
+```
 
-The following update to the `hooks.build` section of `.magento.app.yaml` installs NVM and the latest LTS release of Node.
+### Finding the correct UPWARD path value
 
-It also uses NPX to run the `buildpack` CLI tool with the `create-project` sub-command to scaffold the application using the `venia-concept` template.
-After scaffolding completes, it installs and builds the application and moves the build artifact to a persistent area on the file system.
+The `CONFIG_DEFAULT_WEB_UPWARD_PATH` variable specifies the _absolute_ path to the UPWARD configuration file in the deployed instance.
+If this value is incorrect or not set, the Magento 2 UPWARD connector extension cannot serve your storefront application and your frontend appears broken.
 
-```yaml
-hooks:
-    # We run build hooks before your application has been packaged.
-    build: |
-        set -e
+In the previous example, `/app/pmu35riuj7btw_stg/` is the Magento application root directory on the deployed instance.
+This value is different for each environment in your Cloud project, so you must configure each of your project environments with the path specific to each instance.
+To find the correct root directory path for an environment, [SSH][] into the remote server and use the `pwd` command to find the Magento application root directory.
 
-        unset NPM_CONFIG_PREFIX
-        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.35.1/install.sh | bash
-        export NVM_DIR="$HOME/.nvm"
-        [ -s "$NVM_DIR/nvm.sh"  ] && \. "$NVM_DIR/nvm.sh"
-        nvm install --lts=dubnium
+## Build your storefront application
 
-        npx @magento/pwa-buildpack create-project /tmp/pwa --template venia-concept --backend-url https://[your-cloud-url-here] --name Venia --author Magento --install false --npm-client yarn
-        cd /tmp/pwa
-        # Override this environment variable at runtime given all scaffolded dependencies are devDependencies
-        NODE_ENV=development yarn install --ignore-optional
-        yarn build
-        cp -p -r dist/ "$HOME/pwa"
-        cd $HOME
+Navigate or open a new terminal to _your storefront project_ and edit the `.env` file.
+Set the values for each variable to the same value as the ones you set in the previous step.
 
-        php ./vendor/bin/ece-tools build:generate
-        php ./vendor/bin/ece-tools build:transfer
+Use `yarn` or `npm` to run the project's build command.
```

Comment:
Don't we only support yarn due to the monorepo?

Reply:
if you're developing from the monorepo, then yes. But this is targeted for people who have scaffolded a project, which gives you the option of using npm or yarn as a package manager,